packages/app/src/app/overmind/effects/vscode/utils.ts

```diff
@@ -0,0 +1,64 @@
+import { getModulePath } from '@codesandbox/common/lib/sandbox/modules';
+import { Sandbox } from '@codesandbox/common/lib/types';
+import { lineAndColumnToIndex } from 'app/overmind/utils/common';
+
+export function getVSCodePath(sandbox: Sandbox, moduleId: string) {
+  return `/sandbox${getModulePath(
+    sandbox.modules,
+    sandbox.directories,
+    moduleId
+  )}`;
+}
+
+export function getCurrentModelPath(editor) {
+  const activeEditor = editor.getActiveCodeEditor();
+
+  if (!activeEditor) {
+    return undefined;
+  }
+
+  const model = activeEditor.getModel();
+
+  if (!model) {
+    return undefined;
+  }
+
+  return model.uri.path.replace(/^\/sandbox/, '');
+}
+
+export function getCode(editor) {
+  const activeEditor = editor.getActiveCodeEditor();
+
+  if (!activeEditor) return '';
```

Comment:
Maybe makes sense to return `null` here? Unless we used to return `''` as well, then it's probably better to return the same for compat reasons.

Reply:
It is actually no being used for anything, removed it 😄